src/ol_infrastructure/applications/ocw_studio/__main__.py

```diff
@@ -378,6 +378,13 @@
     "VIDEO_TRANSCODE_QUEUE": ocw_studio_mediaconvert.queue.name,
 }
 
+if stack_info.env_suffix.lower() != "production":
```

Comment:
What's the reasoning for not running against production? Is it because of the proxy?

Reply:
The reasoning here is cost. With the amount of traffic OCW gets, having it on the production site is incurring far more events than we originally thought. I think that we will put up a PR to disable analytics in the client though instead: https://posthog.com/docs/libraries/js#opt-out-of-data-capture. The question right now is surrounding the question of "is the cost still going to be too much with production OCW asking if a feature flag is enabled on every page load?"

Reply:
Rather than putting these in as a conditional, it would be cleaner to just add them to the stack configs. That way it's easier to manage separate values per environment.